cardano_node_tests/tests/plutus_common.py

```diff
@@ -129,6 +129,7 @@ class ExecutionCost:
 )
 
 MINTING_V3_COST = ExecutionCost(per_time=161_100, per_space=800, fixed_cost=58)
+MINTING_V3_REF_COST = ExecutionCost(per_time=161_100, per_space=800, fixed_cost=58)
```

Comment:
Shouldn't we see differences in the costs?

Reply:
The cost is probably slightly higher, because there's one more TxIn in script context. It likely fits into that 15% margin (`frac=0.15`), so the test doesn't fail, but better to get the precise values.

Reply:
I got those costs running calculate_plutus_script_cost with the proper inputs, that's why I thought it was weird.